templates/navigation-bar.twig

```diff
@@ -0,0 +1,129 @@
+                                </span>
+                                <div class="mobile-menus">
+                                    <ul class="list-unstyled">
+                                        <li>
+                                            <img class="country-icon" src="{{ data_nav_bar.images }}country-icon.png" alt=""/>
+                                            <div class="select-country styled-select">
+                                                <select>
+                                                    <option selected disabled>- {{ __( 'Select a website', domain ) }} -</option>
+                                                    {% for key, website in data_nav_bar.websites %}
+                                                        {% if key == settings.p4en_lang %}
+                                                            <option value="{{ key }}" selected>{{ website }}</option>
+                                                        {% else %}
+                                                            <option value="{{ key }}">{{ website }}</option>
+                                                        {% endif %}
+                                                    {% endfor %}
+                                                </select>
+                                            </div>
+                                        </li>
+                                        <li class="nav-item active">
+                                            <a href="#">{{ __( 'ACT', domain ) }}</a>
+                                        </li>
+                                        <li class="nav-item">
+                                            <a href="#">{{ __( 'EXPLORE', domain ) }}</a>
+                                        </li>
+                                    </ul>
+                                    <a href="#" class="btn btn-secondary">{{ __( 'DONATE', domain ) }}</a>
+                                </div>
+                            </div>
+                            <div class="col">
+                                <a class="navbar-brand" href="#"><img src="{{ data_nav_bar.images }}greenpeace-logo.png" alt="greenpeace" /></a>
+                            </div>
+                            <div class="col text-right">
+                                <div id="search" class="mobile-search-wrap">
+                                    <div class="search-link">
+                                        <a href="#search" id="search-trigger"><i class="fa fa-search fa-lg"></i></a>
+                                    </div>
+                                    <div class="search-box">
+                                        <form action="/search" id="search_form">
+                                            <input placeholder="{{ __( 'Search', domain ) }}" type="text" />
+                                            <input id="search-submit" type="submit" class="btn btn-green" value="Search" />
```

Comment:
Does `Search` in the `value` attribute need to be translated?

Reply:
Not really, since we are using the placeholder (but still I can add it for consistency).